### ISSUE TITLE
Unable to generate xsl documentation under Windows

### ISSUE THREAD

**Issue by cmarchand** [cmarchand/xslDoc-maven-plugin#4]
Using xslDoc maven plugin under windows, I have this error : 
```
[INFO] 2017-01-18 09:28:35,791 ERROR [main] fr.efl.chaine.xslt.GauloisPipe     - C:\Users\EXT-CM~1\AppData\Local\Temp\gp-155851657394446422.xml not found or can not be open
[INFO] fr.efl.chaine.xslt.InvalidSyntaxException: C:\Users\EXT-CM~1\AppData\Local\Temp\gp-155851657394446422.xml not found or can not be open
[INFO] 	at fr.efl.chaine.xslt.config.ConfigUtil.<init>(ConfigUtil.java:85)
[INFO] 	at fr.efl.chaine.xslt.GauloisPipe.parseConfig(GauloisPipe.java:932)
[INFO] 	at fr.efl.chaine.xslt.GauloisPipe.parseCommandLine(GauloisPipe.java:905)
[INFO] 	at fr.efl.chaine.xslt.GauloisPipe.main(GauloisPipe.java:799)
[INFO] 	at top.marchand.xml.xsl.doc.GauloisPipeRunner.main(GauloisPipeRunner.java:28)`
````

Christophe

**Comment by cmarchand** [comment]
Under Windows, there with a windows path file.
Under Linux, there is a linux path file.
gaulois-pipe expect a config URI, and not a path file

**Comment by cmarchand** [comment]
Other problem, when basedir parameter contains a \, the absoluteRoot Folder URL contains a \, and it is not allowed.

**Comment by cmarchand** [comment]
Solved in 0.7
